src/components/Choice/src/ChoiceOption.vue

```diff
@@ -32,6 +45,19 @@ export default {
 			}
 			return this.controlState.currentValue === this.value;
 		},
+
+		style() {
```

Comment:
please move this to the Choice parent component, these styles only have to be calculated once and can be inherited by every ChoiceOption component via CSS inheritance

Reply:
@pretzelhammer Just moved this logic to `getSelectedStyle` in the Choice parent component

Reply:
correct me if i'm wrong, but i don't think this computed prop is necessary on the children components, all of the passed styles are already CSS custom properties so you can just set them on the parent and have the children inherit them naturally, i don't think a `getSelectedStyle` method is necessary

Reply:
@pretzelhammer No you are correct, I was overthinking this. Just made this change